File: _Internal/Model_NorthwindIB_NH/Entities/Region.cs
namespace Models.NorthwindIB.NH
{
    using System;
    using System.Collections.Generic;

    public partial class Region
    {
        public Region()
        {
            this.Territories = new HashSet<Territory>();
        }

        public virtual int RegionID { get; set; }
        public virtual string RegionDescription { get; set; }
        public virtual int RowVersion { get; set; }

        public virtual ICollection<Territory> Territories { get; set; }
    }
}

File: Samples_Unpublished/Java/NorthBreeze/WEB-INF/src/northwind/model/Region.java
package northwind.model;
import java.util.Set;

public class Region {
    private int regionID;
    private String regionDescription;
    private int rowVersion;

    private Set<Territory> territories;

	public int getRegionID() {
		return regionID;
	}

	public void setRegionID(int regionID) {
		this.regionID = regionID;
	}

	public String getRegionDescription() {
		return regionDescription;
	}

	public void setRegionDescription(String regionDescription) {
		this.regionDescription = regionDescription;
	}

	public int getRowVersion() {
		return rowVersion;
	}

	public void setRowVersion(int rowVersion) {
		this.rowVersion = rowVersion;
	}

	public Set<Territory> getTerritories() {
		return territories;
	}

	public void setTerritories(Set<Territory> territories) {
		this.territories = territories;
	}

}
